rust/src/lib/nispor/base_iface.rs

```diff
@@ -26,6 +26,7 @@ fn np_iface_type_to_nmstate(
         nispor::IfaceType::Vxlan => InterfaceType::Vxlan,
         nispor::IfaceType::Ipoib => InterfaceType::InfiniBand,
         nispor::IfaceType::Tun => InterfaceType::Tun,
+        nispor::IfaceType::Other(v) if v == "xfrm" => InterfaceType::Xfrm,
```

Comment:
why not support the `xfrm` interface in nispor ?

Reply:
This is urgent case, release new version if nispor take times.

Reply:
I am creating a JIRA card to track this work :)